src/main/java/com/recruitment/project/parking/parkingspaces/service/UserParkingVisitServiceImpl.java

```diff
@@ -0,0 +1,124 @@
+package com.recruitment.project.parking.parkingspaces.service;
+
+import com.recruitment.project.parking.parkingspaces.domain.UserParkingVisit;
+import com.recruitment.project.parking.parkingspaces.domain.UserVehicle;
+import com.recruitment.project.parking.parkingspaces.repository.UserParkingVisitRepository;
+import com.recruitment.project.parking.parkingspaces.repository.UserVehicleRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static java.lang.Math.round;
+
+
+@Service
+public class UserParkingVisitServiceImpl implements UserParkingVisitService {
+
+    private static final Logger logger = LoggerFactory.getLogger(UserParkingVisitServiceImpl.class);
+
+    @Autowired
+    private UserParkingVisitRepository userParkingVisitRepository;
+
+    @Autowired
+    private UserVehicleRepository userVehicleRepository;
+
+    @Override
+    public List<UserParkingVisit> getAllUserParkingVisit() {
+        return userParkingVisitRepository.findAll();
```

Comment:
jak odniesiemy epicki sukces z tą aplikacją to czy wyjęcie całej bazy danych to dobry pomysł?

Reply:
to była pierwsza funkcja, przeoczyłem jej usunięcie